### PR TITLE
Create proc_creation_win_susp_ping_del.yml

### PR DESCRIPTION
**Pull request by krestinichev**
Add new rule

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_susp_ping_del.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: Suspicious activity similar to ransomware
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_ping_del.yml:1]:
```suggestion
title: Suspicious Ping And Del Combination 
```

File: rules/windows/process_creation/proc_creation_win_susp_ping_del.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    product: windows
+detection:
+    selection:
+        CommandLine|contains|all:
+            - 'ping'
+            - '-n'
+            - '& del'
+    condition: selection
+falsepositives:
+    - Unknown
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_ping_del.yml:25]:
```suggestion
    - False positive could occur in admin scripts that execute inline
```

File: rules/windows/process_creation/proc_creation_win_susp_ping_del.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    - attack.t1070.004
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains|all:
+            - 'ping'
+            - '-n'
+            - '& del'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_ping_del.yml:22]:
```suggestion
    selection:
        CommandLine|contains|all:
            # Example: "C:\Windows\System32\cmd.exe"  /C ping 127.0.0.7 -n 3 > Nul & fsutil file setZeroData offset=0 length=524288 "C:\Users\User\Desktop\lockbit\lockbit.exe" & Del /f /q "C:\Users\User\Desktop\lockbit\lockbit.exe".
            - 'ping '
            - ' -n'
            - '& del'
```

File: rules/windows/process_creation/proc_creation_win_susp_ping_del.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    condition: selection
+falsepositives:
+    - Unknown
+level: high
```
[the remark this answers is not in the document]
Comment (frack113) [reply rules/windows/process_creation/proc_creation_win_susp_ping_del.yml:28]: Some test on windows 11
First part is the wait
- ping or ping.exe  # the two works
- '-n ' or '/n ' #need the space after then n
- ' >nul' or ' > nul'
- '&' # we launch another command

Second part is the critical action to have a `high` level
here it is a simple del , can be a powershell , vbs too , no?

for the del miss the parameters force and quiet mode eq `/f` and `/q`.

File: rules/windows/process_creation/proc_creation_win_susp_ping_del.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains|all:
+            # Example: "C:\Windows\System32\cmd.exe"  /C ping 127.0.0.7 -n 3 > Nul & fsutil file setZeroData offset=0 length=524288 "C:\Users\User\Desktop\lockbit\lockbit.exe" & Del /f /q "C:\Users\User\Desktop\lockbit\lockbit.exe".
+            - 'ping '
+            - ' -n'
+            - '& del'
+    condition: selection
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_ping_del.yml:25]:
```suggestion
detection:
    # Example: "C:\Windows\System32\cmd.exe"  /C ping 127.0.0.7 -n 3 > Nul & fsutil file setZeroData offset=0 length=524288 "C:\Users\User\Desktop\lockbit\lockbit.exe" & Del /f /q "C:\Users\User\Desktop\lockbit\lockbit.exe".
    selection_count:
        CommandLine|contains:
            - ' -n '
            - ' /n '
    selection_nul:
        CommandLine|contains: 'Nul' # Covers "> Nul" and ">Nul "
    selection_del_param:
        - ' /f '
        - ' -f '
        - ' /q '
        - ' -q '
    selection_all:
        CommandLine|contains|all:
            - 'ping' # Covers "ping" and "ping.exe"
            - 'del '
    condition: all of selection_*
```

Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_susp_ping_del.yml:25]: @frack113 check this suggestion and let me know what you think.

Comment (frack113) [reply rules/windows/process_creation/proc_creation_win_susp_ping_del.yml]: Seen good for me

**Review by frack113 — APPROVED** [review]

**Pull request merged**